### PR TITLE
CRS-2061 Show SDS exclusions for single sentence

### PR DESCRIPTION
**Pull request by FaizJD030**
https://dsdmoj.atlassian.net/browse/CRS-2061

### CODE REVIEW

File: server/routes/checkInformationRoutes.test.ts [hunk 1]
```diff
@@ -555,6 +555,57 @@ describe('Check information routes tests', () => {
...
+    return request(app)
+      .get('/calculation/A1234AA/check-information')
+      .expect(200)
+      .expect('Content-Type', /html/)
+      .expect(res => {
+        const $ = cheerio.load(res.text)
+        expect($('.new-sentence-card:contains("VIOOFFENCE")').text()).toContain('Violent')
+        const noExclusionCard = $('.new-sentence-card:contains("No exclusion offence")')
+        expect(noExclusionCard.text()).not.toContain('Sexual')
+        expect(noExclusionCard.text()).not.toContain('Violent')
```
Comment (dave-virden) [inline server/routes/checkInformationRoutes.test.ts:605]: I don't think you need these lines 603-5

**Review by dave-virden — COMMENTED** [review]
There's also the genuine overrides sentenceCard which might need similar attention (currently it's not tested with a single offence)

File: server/routes/checkInformationRoutes.test.ts (hunk 1 above)
Comment (FaizJD030) [reply server/routes/checkInformationRoutes.test.ts]: Amended as requested.

**Comment by FaizJD030** [comment]
> There's also the genuine overrides sentenceCard which might need similar attention (currently it's not tested with a single offence)

Addressed Genuine overrides as requested.

**Review by dave-virden — APPROVED** [review]
LGTM

**Pull request merged**